Logger/FileLogger.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace Logger;
+using System;
+using System.IO;
+using System.Globalization;
+
+public class FileLogger (string? fileName) : BaseLogger
+{
+    private string? FilePath { get; set; } = fileName;
+
+    private readonly string? filePath = fileName;
+
+    public override void Log(LogLevel logLevel, string message)
+    {
+        if (filePath == null)
+        {
+            throw new ArgumentNullException(nameof(logLevel), " File path can not be null");
+        }
+        DateTime date = DateTime.Now;
+        string currentDate = date.ToString("MM-dd-yyyy HH:mm:ss tt", CultureInfo.CurrentCulture);
+        string m = $"{currentDate} {nameof(FileLogger)} {logLevel}: {message}";

```

Comment:
I believe this should just be ClassName and not name0f(FileLogger) since whoever is making the LogFactory will set the ClassName when creating their LogFactory.
```suggestion
        string m = $"{currentDate} {this.ClassName} {logLevel}: {message}";

```